app/src/main/res/drawable/thumb_logmein.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="446dp"
```

Comment:
We try to keep thumbnails bellow 240dp

Reply:
Oops, I thought the shorter side was supposed to be <240dp. Will fix.

Reply:
Fixed.